tests/run-travis.sh

```diff
@@ -11,14 +21,14 @@ function run_packages_tests {
 			export NAME=$(basename $(pwd))
 
 			if [ ! -e tests/php/travis-can-run.sh ] || tests/php/travis-can-run.sh; then
-				if [ "$DO_COVERAGE" == "true" ]; then
-					composer install
+				if [ "$DO_COVERAGE" == "true" ] && [[ "$NAME" != "codesniffer" ]]; then
+					composer install -q
 					export WP_TRAVISCI_PACKAGES="phpdbg -d memory_limit=2048M -d max_execution_time=900 -qrr ./vendor/bin/phpunit --coverage-clover $TRAVIS_BUILD_DIR/coverage/packages/$NAME-clover.xml"
 				fi
 				echo "Running \`$WP_TRAVISCI_PACKAGES\` for package \`$NAME\` "
 
 				if $WP_TRAVISCI_PACKAGES; then
-					ls -la $TRAVIS_BUILD_DIR/coverage/packages
+					ls -la $TRAVIS_BUILD_DIR/coverage/
```

Comment:
This too only makes sense when DO_COVERAGE=true.

If it were reached, the change you made to this line would make it kind of pointless anyway. Before it would have shown the sizes of the generated coverage XML files. Now it would just print something useless along the lines of
```
total 12
drwxrwxr-x  3 travis travis 4096 Nov  9 15:54 .
drwxrwxr-x 24 travis travis 4096 Nov  9 15:54 ..
drwxrwxr-x  2 travis travis 4096 Nov  9 15:54 packages
```
That also applies to line 106 below.

Reply:
Removed in 4000664